links.html

```diff
@@ -58,26 +59,10 @@
         </nav>
       </header>
 
-      <div class="Links">
-        <main>
-          <a href="/html/default.asp">HTML tutorial</a>
-          <h1>Useful Links</h1>
-          <ul>
-            <li>
-              <a href="https://validator.w3.org/nu/" target="_blank"
-                >W3 Validator</a
-              >
-            </li>
-            <li>
-              <a href="https://realfavicongenerator.net/" target="_blank"
-                >Favicon Generator</a
-              >
-            </li>
-
+      <div class="width-wrap">
 
       <main>
        
-        <a href="/html/default.asp">HTML tutorial</a>
```

Comment:
Merging problems.  Looks like you kept parts of both your and my changes, causing duplicate code